circuit_knitting/cutting/cut_finding/best_first_search.py

```diff
@@ -299,10 +314,10 @@ def minimum_reached(self) -> bool:
         """Return True if the optimization reached a global minimum."""
         return self.min_reached
 
-    def get_stats(self, penultimate: bool = False) -> np.typing.NDArray[np.int_] | None:
+    def get_stats(self, penultimate: bool = False) -> NamedTuple | None:
```

Comment:
Does it complain if you use `SearchStats` as the return type?

Reply:
It doesn't, I have changed that here ([d931ec6](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/585/commits/d931ec60b33e730e290111c973552ed8c7111af2)).